examples/hello_world.py

```diff
@@ -1,11 +1,12 @@
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
 import twingly_search
-import datetime
 
 client = twingly_search.Client()
-query = client.query()
-query.pattern = '"hello world"'
-query.start_time = datetime.datetime.utcnow() - datetime.timedelta(hours=1)
-results = query.execute()
+q = '"hello world" tspan:24h'
+results = client.execute_query(q)
```

Comment:
Ignore this comment, check the conversation at https://github.com/twingly/twingly-search-api-python/pull/32/files/1e3a4f6b18b4b043e227c6c4020d827720f1104f#r117046935 :)